pina/label_tensor.py

```diff
@@ -96,7 +97,12 @@ def clone(self, *args, **kwargs):
         :return: a copy of the tensor
         :rtype: LabelTensor
         """
-        return LabelTensor(super().clone(*args, **kwargs), self.labels)
+        try:
+            out = LabelTensor(super().clone(*args, **kwargs), self.labels)
+        except:
+            out = super().clone(*args, **kwargs)
```

Comment:
not really understandable: why do we need the exception?
Add the exception please

Reply:
This is tricky, basically lightining during the optimization procedure calls a clone and detach. The problem is that the loss is a one dimensional vector (e.g. `torch.tensor([10])`), hence we cannot assign a label to it. Even if we try to assign a label, it is lost during the lightining optimization face. This is just a temporal workaround.

Personally, I would avoid the try and except routine, since it is pretty dangerous. **The best thing would be to extract the torch.tensor vector from the loss (so do not keep the Label tensor), and return this at the end of the training step in Solver.** In this way we do not need the try and except in clone, and we can be sure that lightining is doing is job. It is important to keep gradient information tho.

Reply:
We haven't solved the issue, a comment is put to remind us to do it in the future